lib/updateVersion.js

```diff
@@ -480,7 +481,7 @@ function objectsToArrays(gltf) {
     });
     ForEach.animation(gltf, function(animation) {
         var samplerMapping = {};
-        animation.samplers = objectToArray(animation.samplers, samplerMapping);
+        animation.samplers = objectToArray(animation.samplers, samplerMapping, 'samplers');
```

Comment:
It would be cleaner to add a separate function called `removeAnimationSamplerNames` and called after `objectsToArrays`.

From the spec it looks look `gltf.samplers` can have names but `animation.samplers` cannot.